python/src/com/jetbrains/python/psi/types/PyImportedModuleType.java

```diff
@@ -43,20 +37,33 @@ public PyImportedModuleType(@NotNull PyImportedModule importedModule) {
     final PsiElement resolved = myImportedModule.resolve();
     if (resolved != null) {
       final PsiFile containingFile = location != null ? location.getContainingFile() : null;
-      List<PsiElement> elements = Collections.singletonList(ResolveImportUtil.resolveChild(resolved, name, containingFile, false, true,
-                                                                                           false));
+      PsiElement resolvedChild = ResolveImportUtil.resolveChild(resolved, name, containingFile, false, true,
+                                                                false);
```

Comment:
I'd use `resolveChildren` instead.

Reply:
Yes, that might fix https://youtrack.jetbrains.com/issue/PY-28393